helm/alfresco-content-services/6.0.N_values.yaml

```diff
@@ -0,0 +1,619 @@
+    externalPort: 80
+  resources:
+    requests:
+      memory: "1000Mi"
+    limits:
+      memory: "1000Mi"
+  environment:
+    JAVA_OPTS: " -Xms256M -Xmx512M"
+  readinessProbe:
+    initialDelaySeconds: 20
+    periodSeconds: 60
+    timeoutSeconds: 10
+  livenessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 20
+    timeoutSeconds: 10
+    livenessPercent: 150
+    livenessTransformPeriodSeconds: 600
+    maxTransforms: 10000
+    maxTransformSeconds: 1200
+
+# Declares the alfresco-imagemagick service used by the content repository
+# to transform image files
+imagemagick:
+  replicaCount: 2
+  image:
+    repository: alfresco/alfresco-imagemagick
+    tag: "1.2"
+    pullPolicy: Always
+    internalPort: 8090
+  service:
+    name: imagemagick
+    type: ClusterIP
+    externalPort: 80
+  resources:
+    requests:
+      memory: "1000Mi"
+    limits:
+      memory: "1000Mi"
+  environment:
+    JAVA_OPTS: " -Xms256M -Xmx512M"
+  readinessProbe:
+    initialDelaySeconds: 20
+    periodSeconds: 60
+    timeoutSeconds: 10
+  livenessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 20
+    timeoutSeconds: 10
+    livenessPercent: 150
+    livenessTransformPeriodSeconds: 600
+    maxTransforms: 10000
+    maxTransformSeconds: 900
+
+# Declares the alfresco-libreoffice service used by the content repository
+# to transform office files
+libreoffice:
+  replicaCount: 2
+  image:
+    repository: alfresco/alfresco-libreoffice
+    tag: "1.2"
+    pullPolicy: Always
+    internalPort: 8090
+  service:
+    name: libreoffice
+    type: ClusterIP
+    externalPort: 80
+  resources:
+    requests:
+      memory: "1000Mi"
+    limits:
+      memory: "1000Mi"
+  environment:
+    JAVA_OPTS: " -Xms256M -Xmx512M"
+  readinessProbe:
+    initialDelaySeconds: 20
+    periodSeconds: 60
+    timeoutSeconds: 10
+  livenessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 20
+    timeoutSeconds: 10
+    livenessPercent: 250
+    livenessTransformPeriodSeconds: 600
+    maxTransforms: 99999
+    maxTransformSeconds: 1800
+
+# Declares the alfresco-tika service used by the content repository
+# to transform office files
+tika:
+  replicaCount: 2
+  image:
+    repository: quay.io/alfresco/alfresco-tika
+    tag: "2.0.17"
+    pullPolicy: Always
+    internalPort: 8090
+  service:
+    name: tika
+    type: ClusterIP
+    externalPort: 80
+  resources:
+    requests:
+      memory: "1000Mi"
+    limits:
+      memory: "1000Mi"
+  environment:
+    JAVA_OPTS: " -Xms256M -Xmx512M"
+  readinessProbe:
+    initialDelaySeconds: 20
+    periodSeconds: 60
+    timeoutSeconds: 10
+  livenessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 20
+    timeoutSeconds: 10
+    livenessPercent: 400
+    livenessTransformPeriodSeconds: 600
+    maxTransforms: 10000
+    maxTransformSeconds: 1800
+
+# Declares the alfresco-tika service used by the content repository
+# to transform office files
+transformmisc:
+  enabled: false
+  replicaCount: 2
```

Comment:
Do we need to specify all the values below if the "transformmisc.enabled" option is false?

Reply:
nope, removed them here and in 6.1